Dockerfile.template

```diff
@@ -40,6 +40,7 @@ WORKDIR /var/www/html
 {{ ) end -}}
 
 # install the PHP extensions we need (https://make.wordpress.org/hosting/handbook/handbook/server-environment/#php-extensions)
+# Note: imagick uses libheif for AVIF support, whereas GD uses libavif for AVIF support
```

Comment:
The wording of this note had me looking for support for AVIF / HEIF in "imagick" until I found https://github.com/Imagick/imagick/issues/280#issuecomment-484741943 which made it more clear it's _imagemagick_ that uses libheif, not imagick directly. :sweat_smile:

I think we should drop this comment line and make some minor adjustments below to make that relationship more clear.

Reply:
Comment removed in https://github.com/docker-library/wordpress/pull/919/commits/936bbfcaccbfd3e7b9c40c2e0f26b1aa103a8b14